### PR TITLE
OP-1340 Force Lot's cost definition when LOTWITHCOST=yes and AUTOMATICLOT_IN=no

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1340.
Paired with https://github.com/informatici/openhospital-core/pull/1434

### CODE REVIEW

**Review by dbmalkovsky — APPROVED** [review]
LGTM; left a comment that can be ignored

**Comment by mwithi** [comment]
> LGTM; left a comment that can be ignored

Thanks! Which comment?

File: bundle/language_en.properties [hunk 1]
```diff
@@ -846,6 +846,7 @@ angal.medicalstock.multiplecharging.preparationdate
 angal.medicalstock.multiplecharging.qtypacket                                                          = Qty/Packet
 angal.medicalstock.multiplecharging.referencenumberabb                                                 = Reference No.
 angal.medicalstock.multiplecharging.selectedlot                                                        = Selected Lot
+angal.medicalstock.multiplecharging.selectedlotwithoutcostpleasespecify                                = Selected Lot {0} without cost, please specify...
```
Comment (dbmalkovsky) [inline bundle/language_en.properties]: I forgot to save the comment ☹️ and on second thought my alternative wasn't much better (without cost -> has not cost information)

Comment (mwithi) [reply bundle/language_en.properties]: I prefer your by the way, more professional, thanks!

**Review by dbmalkovsky — COMMENTED** [review]
LGTM

**Pull request merged**